src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -22,59 +23,88 @@ export default class ResultsViewPage extends React.Component<IResultsViewPagePro
     constructor(props: IResultsViewPageProps) {
         super();
 
-        const reaction1 = reaction(
-            () => props.routing.location.query,
-            query => {
-
-                const validationResult = validateParameters(query, ['studyId']);
-
-                if (validationResult.isValid) {
-                    resultsViewPageStore.urlValidationError = null;
-
-                    resultsViewPageStore.studyId = query.studyId;
-
-                    if ('sampleListId' in query) {
-                        resultsViewPageStore.sampleListId = query.sampleListId as string;
-                    }
-
-                    // TODO we may want to split by ","
-
-                    if ('sampleList' in query) {
-                        resultsViewPageStore.sampleList = (query.sampleList as string).split(" ");
-                    }
-
-                    if ('geneList' in query) {
-                        resultsViewPageStore.hugoGeneSymbols = (query.geneList as string).split(" ");
-                    }
-                }
-                else {
-                    resultsViewPageStore.urlValidationError = validationResult.message;
-                }
+        this.exposeComponentRenderersToParentScript();
+
+        // const reaction1 = reaction(
```

Comment:
i realize this (and nulling render below) will mean we can't develop in standalone dev mode.  that can be remedied by using the the exposed render function inside the render of this component and then only when we know we are in standalone mode.  there is a flag for that.